kernel/packages/shared/session/getPerformanceInfo.ts

```diff
@@ -15,27 +15,20 @@ export function getPerformanceInfo(data: { samples: string; fpsIsCapped: boolean
     total: sum,
     len: length,
     min: sorted[0],
-    p50: sorted[Math.ceil(length * 0.5)],
+    p1: sorted[Math.ceil(length * 0.01)],
+    p5: sorted[Math.ceil(length * 0.05)],
+    p10: sorted[Math.ceil(length * 0.10)],
+    p20: sorted[Math.ceil(length * 0.20)],
+    p50: sorted[Math.ceil(length * 0.50)],
     p75: sorted[Math.ceil(length * 0.75)],
-    p90: sorted[Math.ceil(length * 0.9)],
+    p80: sorted[Math.ceil(length * 0.80)],
+    p90: sorted[Math.ceil(length * 0.90)],
     p95: sorted[Math.ceil(length * 0.95)],
     p99: sorted[Math.ceil(length * 0.99)],
     max: sorted[length - 1],
-    capped: data.fpsIsCapped
```

Comment:
We are using this bit

Reply:
Sorry for the short review, I'm in mobile. @pravusjif can you confirm we are using this flag to measure two different rendering profiles?

Reply:
Yes, we do

Reply:
@menduz  I have not removed the flag, it appears that way because i added a comma and more data to this event

Reply:
@menduz can you confirm this appears ok from your end? I have not removed the flag
Thanks!

Reply:
Totally my bad, I got confused for the small screen